### PR TITLE
Return info dictionary from `ImageSource.save()`

### LINKED ISSUE #545: Filename of saved `.mrcs` from `ImageSource.save()`
The current behavior of `ImageSource.save()` is that it accepts a STAR filename for output, grabs the stem of this filename, and tacks on `_{first_index}_{last_index}.mrcs`. This is true whether the savemode is "single" or "multiple". 

Following discussion in #520, we may want to have `ImageSource.save()` return these filename(s) and/or change the behavior, at least for "single" savemode.

### PR DESCRIPTION
**Pull request by chris-langfield**
Closes #545 by returning information about the files saved.

### CODE REVIEW

File: src/aspire/source/image.py [hunk 1]
```diff
@@ -583,6 +584,9 @@ def save(
             batch_size=batch_size,
             overwrite=overwrite,
         )
+        # return some information about the saved files
+        info = {"starfile": starfile_filepath, "mrcs": unique_filenames}
+        return info
```
Comment (j-c-c) [inline src/aspire/source/image.py:591]: Maybe add the `return` to the docs.

**Review by j-c-c — COMMENTED** [review]
One comment. Otherwise, LGTM.

File: src/aspire/source/image.py (hunk 1 above)
Comment (chris-langfield) [reply src/aspire/source/image.py:591]: done thanks

**Review by j-c-c — DISMISSED** [review]

File: src/aspire/source/image.py [hunk 2]
```diff
@@ -567,6 +567,8 @@ def save(
         :param batch_size: Batch size of images to query.
         :param save_mode: Whether to save all images in a single or multiple files in batch size.
         :param overwrite: Option to overwrite the output MRCS files.
+        :return: A dictionary containing "starfile"--the path to the saved starfile-- and "mrcs", a
+        list of the saved particle stack .mrcs filenames.
```
Comment (garrettwrong) [inline src/aspire/source/image.py]: Hey, I didn't realize this was happening before Josh pointed it out to me, but I think the line beginning with `list` needs to be indented for Sphinx to parse it.

**Review by garrettwrong — CHANGES_REQUESTED** [review]
One doc string change otherwise lgtm, thanks!

File: src/aspire/source/image.py (hunk 2 above)
Comment (garrettwrong) [reply src/aspire/source/image.py]: Cool, had it checked out and rebuilt the doc, lgtm. Thanks!

**Review by garrettwrong — APPROVED** [review]

**Pull request merged**